jupyterhub/handlers/base.py

```diff
@@ -328,7 +328,7 @@ def _record_activity(self, obj, timestamp=None):
 
         user._auth_refreshed = now
 
-        if auth_info == True:
+        if auth_info:
```

Comment:
This isn't correct. True is a distinct value from a truthy dict.

Reply:
Roger, prevent any chance of a... __bool__ I think? Slipping and passing this test, or any other possible value like a non empty array I think. Let me update this commit.

Reply:
commit dropped, push-force'd